src/components/tabsMessagesFriends/TabsMessagesFriends.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
+import { Tab, Tabs, Box, Typography, Avatar } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import messages from './messages.json'
+
+interface TabPanelProps {
+  children?: React.ReactNode
+  index: number
+  value: number
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index } = props
+
+  return (
+    <div>
+      {value === index && (
+        <Box
+          sx={{
+            padding: '30px 0',
+            height: '67vh',
+            overflowY: 'scroll',
+          }}
+        >
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  )
+}
+
+const TabsMessagesFriends = () => {
+  const [value, setValue] = React.useState(0)
+  const { classes } = useStyles()
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue)
+  }
+  return (
+    <Box>
+      <Tabs value={value} onChange={handleChange} variant="fullWidth">
+        <Tab label="Messages" />
+        <Tab label="Friends" />
+      </Tabs>
+      <TabPanel value={value} index={0}>
```

Comment:
I recommend split up for Messages and Friends components. This one is container for both of them. You should always keep in mind the idea of granularity.

Reply:
Moved code for messages to a separate component